src/anemoi/models/models/encoder_processor_decoder.py

```diff
@@ -103,6 +105,13 @@ def __init__(
             dst_grid_size=self._data_grid_size,
         )
 
+        self.boundings = nn.ModuleList(
```

Comment:
Can you add a comment to explain the these lines please?
e.g.
# Create the bounding objects (e.g. to ensure TP is positive definite)

Reply:
Done